software/turtlebot4_simulator.md

```diff
@@ -10,10 +10,18 @@ Source code is available [here](https://github.com/turtlebot/turtlebot4_simulato
 
 ## Installation
 
-```note
-The `turtlebot4_simulator` metapackage can be installed on a PC running Ubuntu Desktop 20.04 with ROS 2 Galactic or Ubuntu Desktop 22.04 with ROS 2 Humble.
+The recommended way to install the Turtlebot4 simulator is to install the debian metapackage, which is available on:
+* Ubuntu 24.04 with ROS 2 Jazzy
+* Ubuntu 22.04 with ROS 2 Humble
+* Ubuntu 20.04 with ROS 2 Galactic _end-of-life, not recommended_
+
+by running
+```bash
+sudo apt install ros-${ROS_DISTRO}-turtlebot4-simulator ros-${ROS_DISTRO}-irobot-create-nodes
```

Comment:
Can we add sudo apt update and a note about sourcing your ros environment first to ensure that the ROS_DISTRO is set

Reply:
Because the exact command for sourcing the environment is distro-specific, I've reverted this to just have the full command without envars and tabs for galactic/humble/jazzy.  Otherwise describing how to correctly source your environment (which may not actually exist yet if they're installing fresh on a desktop and don't have any ROS packages yet) gets complicated.